client/src/lib/usbstick.ts

```diff
@@ -1,26 +1,33 @@
-export const isAvailable = () => {
+const isAvailable = () => {
   return !!window.kiosk
 }
 
-export const isPresent = async () => {
-  return isAvailable() && (await window.kiosk!.getUsbDrives()).length > 0
+export enum UsbDriveStatus {
+  notavailable = 'notavailable',
+  absent = 'absent',
+  present = 'present',
```

Comment:
nit: to mirror `notAvailable` I know, i know… later. 😄 
```suggestion
  available = 'available',
```